src/backend/distributed/utils/citus_readfuncs.c

```diff
@@ -377,7 +377,8 @@ ReadTask(READFUNC_ARGS)
 	READ_ENUM_FIELD(taskType, TaskType);
 	READ_UINT64_FIELD(jobId);
 	READ_UINT_FIELD(taskId);
-	READ_STRING_FIELD(queryString);
+	READ_STRING_FIELD(queryStringLazy);
```

Comment:
Is it intentional to not have `perPlacementQueryStrings ` here?

Reply:
Yes, that was unintentional. I will create a PR for this.